cpp/memilio/mobility/metapopulation_mobility_instant.h

```diff
@@ -129,6 +129,8 @@ class MigrationParameters
      */
     MigrationParameters(const MigrationCoefficientGroup& coeffs)
         : m_coefficients(coeffs)
+        , m_save_indices(0)
+
```

Comment:
When you update the name above, please do it here as well. 

(And there is an additional newline)